plugins/tekton/src/types/output.ts

```diff
@@ -0,0 +1,88 @@
+import { ECPolicy } from '@aonic-ui/pipelines/dist/esm/types/components/Output/types';
+import { V1Pod } from '@kubernetes/client-node';
+
+import { ComputedStatus, TaskRunKind } from '@janus-idp/shared-react';
+
+export enum ENTERPRISE_CONTRACT_POLICY_STATUS {
```

Comment:
will these type definitions be moved to `@aonic-ui` or?

Reply:
Yes, Eventually these types/and some common utils will be moved to aonic-pipelines package.